src/components/ConversationSettings/ConversationSettingsDialog.vue

```diff
@@ -243,11 +243,6 @@ export default {
 		handleShowSettings({ token }) {
 			this.$store.dispatch('updateConversationSettingsToken', token)
 			this.showSettings = true
-			this.$nextTick(() => {
-				if (this.$refs.linkShareSettings) {
-					this.$refs.linkShareSettings.$el.focus()
```

Comment:
The ref `"linkShareSettings"` should be removed as well .

Reply:
Let's keep it till the next time, as it's not breaking anything. There are couple more things in settings to check and fix later